.github/workflows/ci.yml

```diff
@@ -9,14 +9,14 @@ on:
 
 jobs:
   build:
-    uses: ./.github/workflows/build.yml
+    uses: ./.github/workflows/reusable-build.yml
```

Comment:
Would these reusable workflows still be visible on the actions page if we put them into a subdirectory? E.g. `.github/workflows/shared/reusable-build.yml`

Reply:
I initially tried this, but github actions won't run them anywhere other than directly inside the workflows directory 😞